Parallel-Tests.ps1

```diff
@@ -1,10 +1,11 @@
 param(
     [string[]] $directories,
     [string] $testFilter,
-    [string] $outDir,
```

Comment:
This might be used in the ADO build definition. I'm not 100% positive though.

Reply:
This param is not used anywhere else in this script, which was why it seems like a candidate for removal. 
All the ADO tests linked to GitHub PR validation seem to pass OK, because they used the `Test.cmd` script which calls this `Parallel-Tests.ps1` script. 
Do you have other (internal) builds that call `Parallel-Tests.ps1` directly?

Reply:
I checked - we do simply call Test.cmd. You are right.